tasks.py

```diff
@@ -0,0 +1,97 @@
+"""
+PyInvoke developer task file
+https://www.pyinvoke.org/
+
+These tasks can be run using `invoke <NAME>` or `inv <NAME>` from the project root.
+
+To show all available tasks `invoke --list`
+
+To show task help page `invoke <NAME> --help`
+"""
+import invoke
+
+from scripts import check_type_hint_coverage
+
+
+@invoke.task
+def sort(ctx, path=".", check=False, exclude=None):
+    """Sort module imports"""
+    cmds = ["isort", path, "--profile", "black"]
+    if check:
+        cmds.append("--check-only")
+    if exclude:
+        cmds.extend(["--skip", exclude])
+    ctx.run(" ".join(cmds))
+
+
+@invoke.task
+def fmt(ctx, path=".", sort_=True, check=False, exclude=None):
+    """Run code formatter"""
+    if sort_:
+        sort(ctx, path, check=check, exclude=exclude)
+
+    cmds = ["black", path]
+    if check:
+        cmds.append("--check")
+    if exclude:
+        cmds.extend(["--exclude", exclude])
+    ctx.run(" ".join(cmds))
+
+
+@invoke.task
+def lint(ctx, path="great_expectations/core"):
+    """Run code linter"""
+    ctx.run(" ".join(["flake8", path]))
+
+
+@invoke.task
+def hooks(ctx, changes_only=False, diff=False):
+    """Run pre-commit hooks"""
+    cmds = ["pre-commit", "run"]
+    if diff:
+        cmds.append("--show-diff-on-failure")
+    # used in CI - runs faster and only checks files that have changed
+    if changes_only:
+        cmds.extend(["--from-ref", "origin/HEAD", "--to-ref", "HEAD"])
+    else:
+        cmds.extend(["--all-files"])
+    ctx.run(" ".join(cmds))
+
+
+@invoke.task(iterable=["modules"])
+def types(ctx, modules, coverage=False, install_types=False):
+    """Static Type checking"""
+    modules = modules or [
+        # "checkpoint",  # 78
+        # "cli",  # 237
+        # "core",  # 237
+        "data_asset",  # 1
+        # "data_context",  # 228
+        # "datasource",  # 98
+        "exceptions",  # 2
+        # "execution_engine",  # 111
+        # "expectations",  # 453
+        "jupyter_ux",  # 4
+        # "marshmallow__shade",  # 19
+        "profile",  # 9
+        # "render",  # 87
+        # "rule_based_profiler",  # 469
+        "self_check",  # 10
+        "types",  # 3
+        # "validation_operators", # 47
+        # "validator",  # 46
+    ]
+    # TODO: fix modules with low error count
+    ge_modules = [f"great_expectations/{m}" for m in modules]
+    cmds = [
+        "mypy",
+        *ge_modules,
+        "--ignore-missing-imports",
+        "--follow-imports=silent",
+    ]
+    if install_types:
```

Comment:
Yeah initially had the same thought. I'm open to this change but let me explain my thinking first.

My idea was that the `coverage` check is like a `type-checking-light`, and eventually (possibly many months from now) we will stop using it and exclusively use standard `mypy` type-checking. My thinking is that having two separate type-checking commands would lead to confusion.

Reply:
I can see it both ways. I think 2 distinct steps is clearer (maybe with names like `type-check`, `type-coverage`). We can then wholesale delete the `type-coverage` task when we are ready and don't have to edit this function. I don't feel strongly about this though, either way is fine with me.

Reply:
@billdirks I went with the approach of having 2 separate tasks.
`invoke type-check` & `invoke type-coverage`.
I'll be actually making the initial typing fixes (for our modules with low error counts) in a different PR.